misk-testing/src/main/kotlin/misk/time/FakeClock.kt

```diff
@@ -1,3 +1,7 @@
 package misk.time
 
-typealias FakeClock = wisp.time.FakeClock
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class FakeClock @Inject constructor() : wisp.time.FakeClock()
```

Comment:
I actually prefer this style for wrapping wisp code as we can simply inject this and bind normally